app/components/header.tsx

```diff
@@ -1,11 +1,66 @@
-import { Text, Center } from '@chakra-ui/react';
+import { Text, Box, Heading, Avatar, Button } from '@chakra-ui/react';
+import { Link, Form } from '@remix-run/react';
+
+import { TriangleUpIcon, LockIcon } from '@chakra-ui/icons';
+import { useUser } from '~/utils';
 
 export default function Header() {
+  const user = useUser();
   return (
     <header>
-      <Center>
-        <Text fontSize="2xl">Hi! I am the header</Text>
-      </Center>
+      <Box
+        display="flex"
```

Comment:
You can use the `Flex` component vs. `Box` for this.

Reply:
That's a good call, I forgot they had a flex component